.env.common-local

```diff
@@ -26,3 +26,6 @@ PUBLIC_URL=/
 # Variables for Google Analytics
 REACT_APP_TRACKING_ENABLED=false
 REACT_APP_TRACKING_ID=UA-77033033-31
+
+# This is used for local development to toggle debug mode for Google Analytics
+REACT_APP_GA_DEBUG_MODE=false
```

Comment:
Does it need to be redefined in this file? (`.env.common-local` inherits from `.env`?  Or is there an intermediate file where it's `true`?)

Reply:
I added it here as a clue to a developer that this is where you'd want to change it if you want to affect your local environment. I suppose someone could just edit `.env` instead (I think), but this is where we encourage people to adjust their local environment variables.